client/networks/arbitrum/.gitignore

```diff
@@ -1,3 +1,4 @@
 wwwroot/*.js
 node_modules
 typings
+dist
```

Comment:
If `dist` repository is in `.gitignore`, that repository will not be in GitHub, which will cause an error in "main" and "typings" fields of `package.json` file when importing entities from that package

Reply:
Already fixed in last commit